recipe/build.sh

```diff
@@ -10,7 +10,7 @@
 source activate "${CONDA_DEFAULT_ENV}"
 
 ./autogen.sh
-./configure --with-pic --prefix=$PREFIX
+CXXFLAGS="-02" ./configure --with-pic --prefix=$PREFIX
```

Comment:
This will also override CXXFLAGS which are already set at least for OSX, use `CXXFLAGS="${CXXFLAGS} -O2"`. 

Sadly it seems for me locally on OSX that with `-02` the tests fails.